docs/user_guide/assessment.rst

```diff
@@ -237,17 +237,17 @@ values for the groups:
     :options:  +NORMALIZE_WHITESPACE
 
     >>> mf.difference()
-    tpr       0.2
-    fpr       1.0
+    tpr      0.20
+    fpr      1.00
     sel      0.25
-    count     4.0
-    dtype: object
+    count    4.00
+    dtype: float64
```

Comment:
so we ARE changing things, but I think I like these changes.

Reply:
I'm not sure why that ended up changing. It was a slight surprise.

Reply:
Given Python's approach to typing, I was actually treating that as not-a-change :-)

Reply:
And seeing other little doc issues pop up, this might even be due to a new Pandas release. I wish I knew :-/